### PR TITLE
Fixed `MultiConverter.toFieldValue` tuple typed field support

### PR DESCRIPTION
**Pull request by agroszer**

### CODE REVIEW

File: src/z3c/form/converter.rst [hunk 1]
```diff
@@ -1083,6 +1083,42 @@ If the list is empty, the missing value is returned:
...
+  [u'1', u'2', u'3', u'4']
+
+  >>> conv.toWidgetValue((1, 2, 3, 4))
+  [u'1', u'2', u'3', u'4']
+
+Now, let's look at the reverse. We get a tuple because that's the last
+type in `_type`
+
+  >>> conv.toFieldValue([u'1', u'2', u'3', u'4'])
+  (1, 2, 3, 4)
```
Comment (kedder) [inline src/z3c/form/converter.rst:1119]: Why are you asserting the list is getting converted to a tuple?

Comment (kedder) [reply src/z3c/form/converter.rst:1119]: Oh, didn't notice the comment above. Anyways, why last type in _type is special? Is it a feature or just conincidence?

Comment (agroszer) [reply src/z3c/form/converter.rst:1119]: the list is the input, coming from the request, the return value is the value we want to use in our app -- the type is the last from _type, tuple in this case

Comment (kedder) [reply src/z3c/form/converter.rst:1119]: why the last type in _type is special? Is it a feature or just a conincidence?

Comment (agroszer) [reply src/z3c/form/converter.rst:1117]: "copy what the others do" `TextLinesConverter` and `CollectionSequenceDataConverter` does the same, I'd stick to that convention (fixed the .rst's a bit)

**Review by kedder — APPROVED** [review]
Looks good!

**Pull request merged**